apps/api/src/app/layouts/usecases/delete-layout/delete-layout.use-case.ts

```diff
@@ -34,14 +34,11 @@ export class DeleteLayoutUseCase {
       throw new ConflictException(`Layout with id ${command.layoutId} is being used so it can not be deleted`);
     }
 
-    await this.layoutRepository.deleteLayout(command.layoutId, layout._environmentId, layout._organizationId);
+    if (layout.isDefault) {
+      throw new ConflictException(`Layout is being used as your default layout, so it can not be deleted`);
```

Comment:
```suggestion
      throw new ConflictException(`Layout with id ${command.layoutId} is being used as your default layout, so it can not be deleted`);
```